visual-testing-app/src/components/page-content-container-wide/page-content-container-wide.visualroute.tsx

```diff
@@ -83,7 +83,7 @@ export const Component = () => (
           name: 'two columns 2/1 with big gap',
           path: 'two-columns-two-thirds-big-gap',
```

Comment:
The name of this test should also be changed
```suggestion
          name: 'two columns 2/1 with small gap',
          path: 'two-columns-two-thirds-small-gap',
```

Reply:
Updated here: https://github.com/commercetools/merchant-center-application-kit/pull/2967/commits/860a979efe49a26e94a9cfe903bc990e276dd91a